examples/contour/01-contour-config.yaml

```diff
@@ -6,8 +6,11 @@ metadata:
   namespace: projectcontour
 data:
   contour.yaml: |
-    # determine which XDS Server implementation to utilize in Contour.
-    # xds-server-type: contour
+    #
+    # server:
```

Comment:
need to document this in `site/docs/main/configuration.md` as well

Reply:
just pushed that. =)